be/src/storage/primary_key_encoder.cpp

```diff
@@ -238,9 +241,6 @@ inline Status decode_slice(Slice* src, std::string* dest, bool is_last) {
 }
 
 bool PrimaryKeyEncoder::is_supported(const VectorizedField& f) {
-    if (f.is_nullable()) {
-        return false;
-    }
```

Comment:
Will this affect Primary Key's restriction? so primary key still don't support nulllable yet?

Reply:
> Will this affect Primary Key's restriction? so primary key still don't support nulllable yet?

Yes, the keys of PrimaryKey table should not be nullable.